src/plugins/cookies/cookie.js

```diff
@@ -0,0 +1,38 @@
+import URI from 'URIjs';
+
+export default class ChromeCookie {
+
+    constructor (details) {
+        ChromeCookie.assertParams(details);
+        this.details = details;
+        this.url = details.url;
+    }
+
+    toString () {
+        const domain = new URI(this.details.url).hostname();
+        const data = {
+            name: this.details.name || '',
+            value: this.details.value || '',
+            domain: domain,
+            hostOnly: domain.charAt(0) !== '.',
+            httpOnly: Boolean(this.details.httpOnly),
+            secure: Boolean(this.details.secure),
+            session: !Boolean(this.details.expirationDate),
+            path: this.details.path || (new URI(this.details.url)).path()
```

Comment:
URI.js can throw errors on some urls, so i suggest to move it into helper and wrap with try catch